src/components/NewGestureButton.svelte

```diff
@@ -18,9 +20,9 @@
   }
 </script>
 
-<button
-  class="h-20 w-20 cursor-pointer flex justify-center items-center mt-5 mb-2
-    p-2 border-5 border-primary rounded-full text-center text-primary"
-  on:click={onClick}>
-  <i class="fas fa-plus fa-2x text-black-500" />
-</button>
+<div class="mt-5 mb-2">
```

Comment:
The focus ring for this button is partly cut off on the left where it "overflows" the overflow-y box. We'll take another look at this separately. At least it has a focus style.